plugin.yaml

```diff
@@ -2243,6 +2470,9 @@ relationships:
             interface:
               default: { get_property: [SOURCE, interface] }
 
+  cloudify.gcp.relationships.instance_remove_access_config:
+    derived_from: cloudify.relationships.gcp.instance_remove_access_config
+
 workflows:
 
   discover_and_deploy:
```

Comment:
This is great. However, we also need to do plugin_1_4.yaml and v2_plugin.yaml.
I recommend checking the diff of plugin YAML and those files. Then just copying these changes to those files and then making sure that everything is consistent.